tests/plugins/wysiwygarea/manual/detachedundo.md

```diff
@@ -0,0 +1,14 @@
+@bender-tags: 4.17.0, bug, 4462
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea,toolbar,basicstyles,link,format,sourcearea,elementspath,undo
+
+
+1. Type anything in the editor.
+
+2. Change text formatting. For example to 'Heading 1'.
+
+3. Reattach editor with double click on "Toggle" button.
+
+**Expected:** There are two undo steps.
```

Comment:
It is possible to have more undo steps created in step 1. One step with changing formatting would be more error-proof.